### PR TITLE
Bump the ansible-lint version to 24.7.0

### PR DESCRIPTION
**Pull request by mandar242**
Bump the ansible-lint version to 24.7.0

### CODE REVIEW

**Review by alinabuzachis — APPROVED** [review]

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -25,5 +25,5 @@ jobs:
     steps:
       - uses: ansible-network/github_actions/.github/actions/checkout_dependency@main
 
-      - name: Run ansible-lint
-        uses: ansible/ansible-lint@v6.21.0
+      - name: run-ansible-lint
```
Comment (alinabuzachis) [inline .github/workflows/linters.yml:28]: Why the change of step name?

Comment (mandar242) [reply .github/workflows/linters.yml:28]: Replaced it with new task from https://github.com/marketplace/actions/run-ansible-lint

Comment (mandar242) [reply .github/workflows/linters.yml:28]: image
